crt_portal/cts_forms/templates/forms/complaint_view/show/description.html

```diff
@@ -3,4 +3,8 @@
     <h3 class="complaint-card-heading text-uppercase">Personal Description</h3>
     <p>{{ description|linebreaks }}</p>
   </div>
+  <div class="crt-portal-card__content">
+    <h3 class="complaint-card-heading text-uppercase">Language</h3>
+    <p>{{ data.language }}</p>
```

Comment:
We might want to add something to show the language instead of the language code, I don't think the intake specialists will be familiar with the abbreviations.

Reply:
@LindsayYoung I have removed language code. Updated and pushed to repo yesterday. Please review and merge.